### DISCUSSION TITLE
Copilot and Visual Studio for Mac Preview 17.3?

### DISCUSSION THREAD

**Discussion by undavide** [community/community#18440] [Copilot Conversations]
Hi,
I've seen that Copilot is compatible with Visual Studio 2022 (not VSCode). I'm on a Mac, and I've installed the latest preview available which is 17.3 (build 198) now. 

Copilot doesn't show up among the available Extensions though, is it Windows only for the time being?
Thank you!

**Comment by TureeZhang** [comment]
I'd like to pay for it ,and I love it on Windows Visual Studio 2022 experience, but it's my work computer.
As for personal I bought a Mac , when I trying to use Copilot then find nothing I can do ,saddly Copilot not support ..

Not very willing to use VS Code on Mac cause .net core development experience absoluty more better in Vs for Mac 2022 preview rather than VS code.(debuuger and env hard to config ,I still not able to run my netcore3.1 project  T_T)

Hope and good to hear Copilot support VS for Mac 2022 preview in the future, thank you so mush !

**Reply by Trunksome** [reply]
Yes, I'd love to see Co-Pilot for Visual Studio on Mac too! It's not easy to configure VSCode when developing in C# for Unity, in my experience (especially things like debugging etc).
